packages/camera/camera/example/lib/main.dart

```diff
@@ -30,9 +30,11 @@ IconData getCameraLensIcon(CameraLensDirection direction) {
     case CameraLensDirection.front:
       return Icons.camera_front;
     case CameraLensDirection.external:
-      return Icons.camera;
+    // This enum is from a different package, so a new value could be added at
+    // any time. The example should keep working if that happens.
+    // ignore: no_default_cases
```

Comment:
Does adding this ignore prevent the linter from complaining when a new `CameraLensDirection` is added? I agree that the code should still work, but we would still want an indication that the code needs to be updated.

Reply:
Yes, it does. I thought it made sense to be less strict in an example since it's not critical that we update it, but you're right, this would mean that we would likely never remember to update it which is worse than it causing us to have to jump through some extra hoops when adding an enum (https://github.com/flutter/flutter/issues/89866).

I'll switch to what I did in non-example code and make it have a non-`switch` fallback.